src/mayara_pi.cpp

```
@@ -1477,23 +1479,60 @@ void mayara_pi::SyncLocalServerUrl() {
 // for alerts -- rather than a dialog of our own that has to be dismissed.
 void mayara_pi::PollGuardAlarms() {
   if (!m_client) return;
+  if (!m_alarms_seeded) {
+    // Wait for a real connected snapshot: Alarms() is empty before that
+    // anyway, but seeding on an empty pass would still flip the flag before
+    // the server's actual state has had a chance to arrive.
+    if (!m_client->Connected()) return;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)